ui/utils.py

```diff
@@ -31,7 +33,10 @@ def get_moira_client():
             )
     if errors:
         raise RuntimeError('\n'.join(errors))
-    return Moira(settings.MIT_WS_CERTIFICATE_FILE, settings.MIT_WS_PRIVATE_KEY_FILE)
+    try:
+        return Moira(settings.MIT_WS_CERTIFICATE_FILE, settings.MIT_WS_PRIVATE_KEY_FILE)
+    except Exception as exc:  # pylint: disable=broad-except
+        raise MoiraException('Something went wrong with creating a moira client: %s', str(exc))
```

Comment:
See the exception chaining comment above

Reply:
OK